### PR TITLE
Add source info

### PR DESCRIPTION
**Pull request by jamescyeh**

### CODE REVIEW

File: db/types.go [hunk 1]
```diff
@@ -65,6 +65,11 @@ type Job struct {
 	// required: true
 	SourceMedia string `redis-hash:"source" json:"source"`
 
+	// Source information
+	//
+	// required: false
+	SourceInfo SourceInfo `redis-hash:"sourceinfo,omitempty" json:"sourceInfo,omitempty"`
```
Comment (zsiec) [inline db/types.go:71]: Should the redis-hash key be `sourceInfo` (with capital "I") to match the json key?

File: db/types.go [hunk 2]
```diff
@@ -144,6 +149,31 @@ type StreamingParams struct {
...
+type ScanType string
+
+const (
+	// ScanTypeProgressive represents a progressive scan type
+	ScanTypeProgressive ScanType = "progressive"
+
+	// ScanTypeInterlaced represents a interlaced scan type
+	ScanTypeInterlaced ScanType = "interlaced"
+
+	// ScanTypeUnknown represents an unknown scan type
```
Comment (zsiec) [inline db/types.go:162]: 🙌

File: db/types.go [hunk 3]
```diff
@@ -144,6 +149,31 @@ type StreamingParams struct {
...
+)
+
+// SourceInfo represents basic information about the source that may be of aid to providers
+//
+// swagger:model
+type SourceInfo struct {
+	Width  uint `redis-hash:"width,omitempty" json:"width,omitempty"`
+	Height uint `redis-hash:"height,omitempty" json:"height,omitempty"`
+	// FIXME: FrameRate?  Framerate?
+	Framerate float64  `redis-hash:"framerate,omitempty" json:"framerate,omitempty"`
```
Comment (zsiec) [inline db/types.go:173]: I'm partial to `FrameRate` myself. :)

File: db/types.go (hunk 1 above)
Comment (jamescyeh) [reply db/types.go:71]: in that file it seemed like most of the redis-hask keys were all lowercase, which is why I went all lowercase

File: db/types.go (hunk 3 above)
Comment (jamescyeh) [reply db/types.go:173]: Yeah.  It should be FrameRate.

File: db/types.go (hunk 1 above)
Comment (zsiec) [reply db/types.go:71]: gotcha, sounds like something we can clean up later then :)

**Pull request merged**